ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -261,6 +270,17 @@ public NonBlockingRouterMetrics(ClusterMap clusterMap) {
           dataNodeId.getPort());
       dataNodeToMetrics.put(dataNodeId, new NodeLevelMetrics(metricRegistry, dataNodeName));
     }
+
+    // AdaptiveOperationTracker trackers
+    getBlobLocalColoLatency = metricRegistry.histogram(MetricRegistry.name(GetBlobOperation.class, "LocalColoLatency"));
```

Comment:
There is a metric named `crossColoRequestCount` in the same class - so following that naming to be consistent (there isn't a metric with dc or datacenter in its name).